client/src/store/new-request-module.ts

```diff
@@ -2476,7 +2477,7 @@ export class NewRequestModule extends VuexModule {
       source = null
     }
     if (destination === 'Tabs') {
-      this.mutateName(this.nameOriginal)
```

Comment:
If we don't use this original to restore the name, is this property no longer needed on the module?

Reply:
let me check

Reply:
No need to clear name. Confirmed with Tracey